src/main/java/com/redhat/cloud/policies/app/auth/RbacClient.java

```diff
@@ -27,9 +27,12 @@ public class RbacClient {
      * as if an Exception is thrown, the cache will not store the result.
      * Catching and returning null would end up in the next call directly
      * return null from the cache without retrying the remote call.
+     *
+     * Inventory permissions are also queried to get the list of host groups
+     * that the user has access to.
      */
     @CacheResult(cacheName = "rbac-cache")
     RbacRaw getRbacInfo(String xrhidHeader) {
-        return rbac.getRbacInfo("policies", xrhidHeader);
+        return rbac.getRbacInfo("policies,inventory", 100, xrhidHeader);
```

Comment:
Just a side comment.
100 is a pretty big number to start with, but with users being able to create stuff (as i understand it's possible with the host groups) someone will probably top that number eventually.